# the measurements are in base link/base_footprint of kinova arm
# check in the python file
# in meters
# 
boundary_safety:
    x_min: 0.6
    x_max: 0.9
    y_min: -0.3
    y_max: 0.3
    z_min: 0.005
    z_max: 0.5
